internal/transformer/classes/generators/dockerfilegenerators/net5appdockerfilegenerator.go

```diff
@@ -0,0 +1,249 @@
+/*
+ *  Copyright IBM Corporation 2021
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package dockerfilegenerators
+
+import (
+	"encoding/xml"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/konveyor/move2kube/environment"
+	"github.com/konveyor/move2kube/internal/common"
+	"github.com/konveyor/move2kube/types/qaengine/commonqa"
+	transformertypes "github.com/konveyor/move2kube/types/transformer"
+	"github.com/konveyor/move2kube/types/transformer/artifacts"
+	"github.com/sirupsen/logrus"
+)
+
+// Net5AppDockerfileGenerator implements the Transformer interface
+type Net5AppDockerfileGenerator struct {
+	Config transformertypes.Transformer
+	Env    *environment.Environment
+}
+
+const (
+	csproj             = ".csproj"
```

Comment:
isn't it already being done in other files for dot net 4 and below?

Reply:
I have made changes in the code and now using the structures defined in csproj.go